src/main/resources/BridgeServer2.conf

```diff
@@ -178,6 +178,9 @@ synapse.oauth.url = https://repo-prod.prod.sagebase.org/auth/v1/oauth2/token
 synapse.oauth.client.id = dummy-value
 synapse.oauth.client.secret = dummy-value
 
+# To reverse geocode location of appointments in CRC controller
+crc.geocode.api.key = dummy-value
```

Comment:
Do we need a corresponding infra change to deploy this key to AWS?

Reply:
Yes. https://github.com/Sage-Bionetworks/BridgeServer2-infra/pull/34 (I'm not 100% sure if this is correct, but I believe if I set the appropriate environment variable in all 3 environments, it'll work.)